Libraries/LibWeb/Painting/Paintable.cpp

```diff
@@ -229,24 +229,36 @@ Painting::BorderRadiiData normalize_border_radii_data(Layout::Node const& node,
     auto l_bottom = l_top;
     auto l_left = rect.height();
     auto l_right = l_left;
-    auto s_top = (top_left_radius_px.horizontal_radius + top_right_radius_px.horizontal_radius);
-    auto s_right = (top_right_radius_px.vertical_radius + bottom_right_radius_px.vertical_radius);
-    auto s_bottom = (bottom_left_radius_px.horizontal_radius + bottom_right_radius_px.horizontal_radius);
-    auto s_left = (top_left_radius_px.vertical_radius + bottom_left_radius_px.vertical_radius);
-    CSSPixelFraction f = 1;
-    f = (s_top != 0) ? min(f, l_top / s_top) : f;
-    f = (s_right != 0) ? min(f, l_right / s_right) : f;
-    f = (s_bottom != 0) ? min(f, l_bottom / s_bottom) : f;
-    f = (s_left != 0) ? min(f, l_left / s_left) : f;
-
-    // If f < 1, then all corner radii are reduced by multiplying them by f.
-    if (f < 1) {
```

Comment:
This comment was a quote from the spec text, so it should be kept as per [LibWebPatterns.md#comments](https://github.com/LadybirdBrowser/ladybird/blob/master/Documentation/LibWebPatterns.md#comments).

Similarly since the `f < 1` condition is based on spec-text, I'd argue it should not be inverted here to keep the code more verbatim wrt the spec description.